### ISSUE TITLE
Can rz/a2m Output PWM signal in linux

### ISSUE THREAD

**Issue by saltedFishh** [renesas-rz/rza_linux-4.19_bsp#1]
I want to speed up the development of RZ / A2M through Linux, but need to use PWM signal to control motor. I want to confirm whether RZ / A2M can realize PWM signal output through programming under Linux (RZ / A2M EVB is used)

**Comment by seebe** [comment]
Yes you can realize PWM in RZ/A Linux. Unfortunately there is no PWM driver in the BSP.
But, you can make a simple driver to do that.

Here is an old Application Note that was written for RZ/A1, but you can learn how to make a very simple driver.
http://renesasrulz.com/rz/m/files_linux/3206

Note that one issue is pin setup. In the current 4.19 BSP, all pin set up is done using the Device Tree. But if you set up your PWM pins in u-boot before you boot, you will not have to do it in Linux.
